PatientRegistrationApp/src/components/ScreeningQuestions.vue

```diff
@@ -0,0 +1,320 @@
+              <v-radio-group>
+                <template v-slot:label>
+                  <div><span class="red--text"><strong>* </strong></span>1. Are you feeling sick today?</div>
+                </template>
+              </v-radio-group>
+            </v-row>
+          </v-col>
+          <v-col cols="12" sm="4" md="3" lg="2">
+            <v-row no-gutters align="center" justify="end">
+              <v-radio-group v-model="screeningQ1" row required>
+                <v-radio label="Yes" value="Yes"></v-radio>
+                <v-radio label="No" value="No"></v-radio>
+              </v-radio-group>
+            </v-row>
+          </v-col>
+        </v-row> 
+        <v-row no-gutters>
+          <v-divider></v-divider>
+        </v-row>
+        <v-row no-gutters>
+          <v-col cols="12" sm="8" md="9" lg="10">
+            <v-row no-gutters>
+              <v-radio-group>
+                <template v-slot:label>
+                  <div><span class="red--text"><strong>* </strong></span>2. Have you ever received a dose of COVID-19 Vaccine?</div>
+                </template>
+              </v-radio-group>
+            </v-row>
+          </v-col>
+          <v-col cols="12" sm="4" md="3" lg="2">
+            <v-row no-gutters align="center" justify="end">
+              <v-radio-group v-model="screeningQ2" row required @change="FirstDoseResponse()">
+                <v-radio label="Yes" value="Yes"></v-radio>
+                <v-radio label="No" value="No"></v-radio>
+              </v-radio-group>
+            </v-row>
+          </v-col>
+          <v-col cols="12" sm="6" md="5" lg="4">
+            <v-row no-gutters>
+              <v-radio-group v-show="patientReceivedFirstDose">
+                <template v-slot:label>
+                  <div><span class="red--text"><strong>* </strong></span>Which vaccine product did you receive?</div>
+                </template>
+              </v-radio-group>
+            </v-row>
+          </v-col>
+          <v-col cols="12" sm="6" md="7" lg="8">
+            <v-row no-gutters align="center" justify="end">
+              <v-radio-group v-model="screeningQ2b" row required v-show="patientReceivedFirstDose">
+                <v-radio label="Pfizer" value="Pfizer"></v-radio>
+                <v-radio label="Moderna" value="Moderna"></v-radio>
+				<v-radio label="Other" value="Other"></v-radio>
+				<v-radio label="I don't know" value="I don't know"></v-radio>
+              </v-radio-group>
+            </v-row>
+          </v-col>
+        </v-row>
+        <v-row no-gutters>
+          <v-divider></v-divider>
+        </v-row>
+        <v-row no-gutters>
+          <v-col cols="12">
+            <v-row no-gutters>
+              <v-radio-group>
+                <template v-slot:label>
+                  <div>3. Have you ever had an allergic reaction to:<br>
+					(This would include a severe allergic reaction [e.g. anaphylaxis] that required treatment with epinephrine
```

Comment:
Can we not let the text go into the area where the questions are (I drew a line to show like a psuedo border)

![image](https://user-images.githubusercontent.com/67079030/106232006-9ebd1280-61b8-11eb-9884-b41db03d0432.png)

Reply:
Done